wuzz.go

```diff
@@ -1016,7 +1055,19 @@ func main() {
 
 	initApp(app, g)
 
-	err = app.ParseArgs(g)
+	// load config (must be done *before* app.ParseArgs, as arguments
+	// should be able to override config values). An empty string passed
+	// to LoadConfig results in LoadConfig loading the default config
+	// location. If there is no config, the values in
+	// config.DefaultConfig will be used.
+	app.LoadConfig(configPath)
+
+	err = app.ParseArgs(g, args)
+
+	// Some of the values in the config need to have some startup
+	// behavior associated with them. This is run after ParseArgs so
+	// that command-line arguments can override configuration values.
+	app.InitConfig()
```

Comment:
@asciimoo because `ParseArgs` can modify the config (see the `--timeout` option), so we don't want to apply it until after `ParseArgs` runs. `ParseArgs` technically could edit the `CLIENT.Timeout` directly, but then that value would differ from the value in `a.config`, which seems to me like a fragile situation as development continues - it'd be easy to inadvertently disable the `--timeout` option by applying the `app.config` at a later time for some reason, which seems like a definite possibility in the future (for example, if we allow changing options from within the program at some point).